src/functionalTest/kotlin/kotlinx/kover/test/functional/framework/writer/KoverWriter.kt

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2017-2022 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.kover.test.functional.framework.writer
+
+import kotlinx.kover.api.*
+import kotlinx.kover.test.functional.framework.common.*
+import kotlinx.kover.test.functional.framework.configurator.*
+import kotlinx.kover.test.functional.framework.configurator.TestVerifyConfig
+
+internal fun ScriptAppender.writeKover(kover: TestKoverConfig?) {
+    if (kover == null) {
+        if (engineForced != null) {
+            block("kover") {
+                writeEngine(null)
```

Comment:
I think it's better to drop a comment for why this `if` is needed, because first you check `engineForced != null` and then call `writeEngine(null)` — looks like a typo, maybe `writeEngine(engineForced)` was meant to be used?